pkg/osquery/extract_windows_certs.go

```diff
@@ -0,0 +1,111 @@
+//go:build windows
+// +build windows
+
+package osquery
+
+import (
+	"crypto/x509"
+	"fmt"
+	"log/slog"
+	"syscall"
+	"unsafe"
+)
+
+var (
+	crypt32                     = syscall.NewLazyDLL("crypt32.dll")
+	certOpenSystemStore         = crypt32.NewProc("CertOpenSystemStoreW")
+	certCloseStore              = crypt32.NewProc("CertCloseStore")
+	certEnumCertificatesInStore = crypt32.NewProc("CertEnumCertificatesInStore")
+)
+
+const (
+	CERT_STORE_PROV_SYSTEM      = 0x0000000A
+	CERT_CLOSE_STORE_FORCE_FLAG = 0x00000001
+)
+
+type CERT_CONTEXT struct {
+	EncodingType uint32
+	EncodedCert  *byte
+	Length       uint32
+	CertInfo     uintptr
+	Store        uintptr
+}
+
+// extractSystemCerts extracts CA certificates from Windows system stores
+func extractSystemCerts(slog *slog.Logger) ([]*x509.Certificate, error) {
+	var certs []*x509.Certificate
+
+	// Certificate store names to check
+	storeNames := []string{"ROOT", "CA"}
+
+	for _, storeName := range storeNames {
+		storeCerts, err := extractCertsFromStore(storeName)
+		if err != nil {
+			// Log error but continue with other stores
+			slog.Warn("Warning: Failed to extract certificates from %s store: %v\n", storeName, err)
```

Comment:
I know lint says `WarnContext`, but we actually prefer `slogger.Log` (which also takes a ctx -- can use `context.TODO()` here).

We also prefer to not use interpolation for log messages -- sloglint will probably catch that as well on a future check, or at least it should. The reason is that when we have a consistent log message like "failed to extract certificates from store", a) they're easier to grep for in the codebase and b) our error reporting system can group them.

So here we would want something like:

```golang
slog.Log(context.TODO(), slog.LevelWarn,
	"failed to extract certificates from store",
	"store_name", storeName
	"err", err,
)
```